include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -84,22 +84,23 @@ enum EOverlayType : byte {
 
 table CartesianImpedenceControlMode {
   /// actual stiffness to set rot:[nm/rad]
-  stiffness:EulerPoseParams;
-
+  stiffness:EulerPose;
```

Comment:
Yes, makes sense. Just that additional parsing for each changed parameter will be required on the java side. All parameters in CartesianImpedenceControlMode are optional in that way.